essentials-tp/src/main/kotlin/io/github/nyliummc/essentials/extensions/TPExtension.kt

```diff
@@ -0,0 +1,11 @@
+package io.github.nyliummc.essentials.extensions
+
+import io.github.nyliummc.essentials.api.Essentials
+import io.github.nyliummc.essentials.base.extension.AbstractEssentialsExtension
+import io.github.nyliummc.essentials.commands.CommandHome
+
+class TPExtension(essentials: Essentials) : AbstractEssentialsExtension("teleport", essentials) {
+    init {
+        essentials.registry.registerCommand(CommandHome::register)
+    }
```

Comment:
If we allow this module to be disabled, the commands will still be registered. Maybe add a `registerCommands` method to AbstractEssentialsExtension since it's an impl detail, but straight to `EssentialsExtension is fine`